models/qualtrics__contact.sql

```diff
@@ -54,8 +54,8 @@ agg_distribution_responses as (
     select 
         contact_id,
         source_relation,
-        -- should i include other distribution channels? qr code, social media, personal links, etc?
-        -- should i add avg time-to-open?
+        -- should i include other distribution channels? qr code, social media, personal links, etc? 
```

Comment:
I believe we landed on not needing to include these. If that is the case, feel free to remove the comment.

Reply:
removed!